src/app/components/LabeledInput/LabeledInput.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import styles from './LabeledInput.module.css';
+
+type LabeledInputProps = {
+  label: string;
+  placeholder: string;
+  value: string;
+  type?: string;
+  required?: boolean;
+  onChange: (value: string) => void;
+};
+
+export default function LabeledInput({
+  label,
+  placeholder,
+  value,
+  type,
+  required,
+  onChange,
+}: LabeledInputProps): JSX.Element {
+  return (
+    <div className={styles.form}>
+      <input
+        id="form__input"
+        className={styles.form__input}
+        type={type}
+        placeholder={placeholder}
+        value={value}
+        required={required}
+        onChange={(e) => onChange(e.target.value)}
+      />
+      <label htmlFor="form_input" className={styles.form__label}>
+        {' '}
+        {label}{' '}
+      </label>
+    </div>
```

Comment:
It is misleading that the `className` is `form`, but the element is a `div`. 
I wouldn't use `form` at all, because you are speaking of a labeled input.

The other issue is, that you have an `id`. You like to use this `LabeledInput` multiple times in a form, but an `id` has to be unique. This leads to invalid links between `label` and `input`.
Instead, I recommend moving `input` inside the `label` and remove `id` and `htmlFor`:

```tsx
<label>
  <input />
</label>
```

Now, you don't need the `div` (which was only used because of styling reasons.

The white spaces in the `label` shouldn't be there. If you like to have a gap, consider using a margin/padding/gap here.

Reply:
Try to adjust it. But it destroys my complete styling